recipe/meta.yaml

```diff
@@ -101,16 +101,19 @@ test:
     ] %}
     {% for each_hdf5_lib in hdf5_libs %}
     - test -f $PREFIX/lib/lib{{ each_hdf5_lib }}.a                           # [unix]
-    - test -f $PREFIX/lib/lib{{ each_hdf5_lib }}.dylib                       # [osx]
```

Comment:
@jakirkham I can, but it will fail when executed unless we move the fortran lib into a different list. I'll revert this when I make the delete-between-make-and-make-check patch.